Automattic/jetpack
# Remove unused api.js

Fixes https://github.com/Automattic/jetpack/pull/32095#issuecomment-1663901576

## Proposed changes:

* Use store to retrieve the subscribers instead of a API call.
* Clean up some files and reuse some logic 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to your favorite testing site 
* Add a subscribe button to your post
* See this in the panel. 🚀
<img width="300" alt="Screenshot 2023-08-04 at 13 53 15" src="https://github.com/Automattic/jetpack/assets/790558/9b97cc82-80fd-4cdd-9853-b68cfdceb7a3">
